packages/cf.js/src/provider.ts

```diff
@@ -392,16 +392,14 @@ export class Provider {
     const {
       appInstanceId,
       action,
-      newState,
-      oldState
+      newState
     } = nodeEvent.data as Node.UpdateStateEventData;
     const appInstance = await this.getOrCreateAppInstance(appInstanceId);
     const event = {
       type: EventType.UPDATE_STATE,
       data: {
         appInstance,
         newState,
-        oldState,
```

Comment:
why are you removing `oldState`?

Reply:
The caller should already have the old state, seems redundant to add in the result